ucan-invocation/constants.js

```diff
@@ -0,0 +1,7 @@
+// UCAN protocol
+export const STORE_ADD = 'store/add'
+
+// Metrics
+export const W3_METRICS_NAMES = {
+  STORE_ADD_ACCUM_SIZE: `${STORE_ADD}-accumulated-size`
```

Comment:
Meaning is basically the same but I would have gone for `total` over `accumulated` for succinctness - just wondering if there was a different reason?

Reply:
changed to total everywhere